pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/Account/CreateAccountCpmToken.java

```diff
@@ -69,7 +71,7 @@ private String toJsonString(Map<String, String> map){
         }
 
         stringBuilder.append("}");
-        System.out.println(stringBuilder.toString());
+        System.out.println(stringBuilder);
```

Comment:
Can you help remove this log statement also since we are cleaning up others anyway.